code/api/src/modules/user/types.js

```diff
@@ -19,6 +19,7 @@ const UserType = new GraphQLObjectType({
     state: { type: GraphQLString },
     zip: { type: GraphQLString },
     country: { type: GraphQLString },
+    twitter: { type: GraphQLString },
```

Comment:
Twitter seems like part of the extensions right? Is this a handle? BE hasn't done anything with regards to twitter as we don't have a story issue created for it and it wasn't in the API contracts